code/modules/reagents/chemistry/reagents/blob_reagents.dm

```diff
@@ -470,7 +470,7 @@
 	reac_volume = ..()
 	var/turf/open/T = get_turf(M)
 	if(istype(T) && prob(reac_volume))
-		T.MakeSlippery(TURF_WET_WATER, min_wet_time = 10 SECONDS, wet_time_to_add = 5 SECONDS)
+		T.MakeSlippery(TURF_WET_LUBE, min_wet_time = 10 SECONDS, wet_time_to_add = 5 SECONDS)
 		M.adjust_fire_stacks(-(reac_volume / 10))
 		M.ExtinguishMob()
 	M.apply_damage(0.4*reac_volume, BRUTE)
```

Comment:
I'd change it to 0.5 maybe? Most of the blob's damage is temporary so if someone pulls out the victim before they crit they're usually fine (which almost always happens unless they're attacking from a lone angle)